### DISCUSSION TITLE
Is There An Option to Open Published Pages in New Tag

### DISCUSSION THREAD

**Discussion by elizaodie** [GitbookIO/community#332] [0. General bugs and questions] [answered]
Not sure if I am doing something wrong, but when I try to Ctrl + Click to open published pages in a new tab, it results in an error. I also noticed that including target=blank in the URL does not work. Is there a way to open pages in a new tab?

**Comment by taszap** [comment, marked as answer]
Hi @elizaodie, 

You should be able to open published pages in a new tab without a problem so this looks like a bug. We would need a bit more information to debug this so if you could please drop us an email (if you haven't already) on support@gitbook.com we would love to help you!
